src/main/config/messages_template.xml

```diff
@@ -423,6 +423,11 @@ See the accompanying LICENSE file for applicable license.
     <response></response>
   </message>
 
+  <message id="DOTJ084E" type="ERROR">
+    <reason>Couldn’t read '%1' with the %2 character set.</reason>
```

Comment:
Curious, are there any illegal characters that will throw this message, which then break our log presentation because the logging can't properly handle it?

Reply:
ah nevermind for a moment I thought `%2` was the character, obviously it's the character set name